src/main/java/io/curity/identityserver/plugin/OptInMFA/MissingAuthenticatorAcrException.java

```diff
@@ -0,0 +1,24 @@
+/*
+ *  Copyright 2020 Curity AB
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ */
+package io.curity.identityserver.plugin.OptInMFA;
+
+public class MissingAuthenticatorAcrException extends RuntimeException
+{
+    public MissingAuthenticatorAcrException()
+    {
+        super("`secondFactor` parameter is required.");
```

Comment:
This message seems to differ from what the type of the Exception describes?

Reply:
True. I renamed the class.